File: ungs-docsys-ms/src/main/java/com/ungs/docsys/dtos/RequirementResponseDto.java
package com.ungs.docsys.dtos;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class RequirementResponseDto {
    private Long id;
    private String description;
    private RequirementTypeResponseDto requirementType;
    private String operator;
    private String expectedValue;
    private Boolean active;
    private LocalDateTime createdDate;
    private LocalDateTime updatedDate;
    private RequirementTargetComparatorResponseDto requirementTargetComparator;
}

File: ungs-docsys-bff/src/modules/job-applications/requirements/dtos/requirement-response.dto.ts
import { RequirementTargetComparatorResponseDto } from "../../requirement-target-comparators/dtos/requirement-target-comparator-response.dto";
import { RequirementTypeResponseDto } from "../../requirement-types/dtos/requirement-type-response.dto";

export class RequirementResponseDto {
    id: number;
    description: string;
    requirementType: RequirementTypeResponseDto;
    operator: string;
    expectedValue: string;
    active: boolean;
    createdDate: Date;
    updatedDate: Date;
    requirementTargetComparator: RequirementTargetComparatorResponseDto;
  }